.github/workflows/build-release-alpine-nodejs.yml

```diff
@@ -34,22 +46,29 @@ jobs:
     - name: Upload alpine node.js
       uses: actions/upload-artifact@v3
       with:
-        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}
-        path:  ${{runner.temp}}/alpine_node/node-${{github.event.inputs.NodeVersion}}-alpine-x64.tar.gz
+        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}_${{matrix.arch}}
+        path:  ${{runner.temp}}/alpine_node/node-${{github.event.inputs.NodeVersion}}-alpine-${{matrix.arch}}.tar.gz
 
   test:
     name: Test node.js ${{github.event.inputs.NodeVersion}}
     needs: [build]
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - arch: x64
+            runs-on: ubuntu-latest
+          - arch: arm64
+            runs-on: ubuntu-latest # TODO: Add and use an ARM64 runner
```

Comment:
Linux arm64 runners are now available in Public Preview: https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/